index.d.ts

```diff
@@ -39,7 +33,7 @@ export interface Options extends RequestInit {
 	/**
 	Shortcut for sending JSON. Use this instead of the `body` option.
 	*/
-	json?: JSONStringifyable;
```

Comment:
This just made the option harder to use in practice. `JSON.stringify()` accepts `unknown` too. And it didn't account for when your have custom types with a `toJSON()` method. So all in all, the strict type had very little value.